pkg/token/filecache.go

```diff
@@ -229,51 +229,50 @@ func (f *FileCacheProvider) Retrieve() (credentials.Value, error) {
 	if !f.cachedCredential.IsExpired() {
 		// use the cached credential
 		return f.cachedCredential.Credential, nil
-	} else {
-		_, _ = fmt.Fprintf(os.Stderr, "No cached credential available.  Refreshing...\n")
-		// fetch the credentials from the underlying Provider
-		credential, err := f.credentials.Get()
-		if err != nil {
-			return credential, err
+	}
```

Comment:
This file change just seems removing the unnecessary else part and leaving all code outside.

Is there any errors or fix we have with this PR ?